docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -97,93 +159,120 @@ Windows::
+==== Update the Configuration
 
-=== Step 3. Start the Target Cluster
+You must update the following configuration:
 
-The target cluster must be of version Hazelcast 5.3.2 or newer.
+* The cluster information
+* The data structure information
 
-Run the following Docker command to start a one-member target cluster.
+To update the cluster information, complete the following steps:
 
-[source,shell]
-----
-docker run -e HZ_NETWORK_PORT_PORT=5901 -p 127.0.0.1:5901:5901 
-           -e HZ_LICENSEKEY="<license>" <1>
-           -e HZ_CLUSTERNAME=target hazelcast/hazelcast-enterprise:{full-version}-slim <2>
-----
-<1> If you don't have a Hazelcast Enterprise license key, you can either use a Hazelcast Viridian https://docs.hazelcast.com/cloud/free-trial[trial cluster] or request a xref:getting-started:enterprise-overview.adoc#step-2-get-your-enterprise-license[trial license].
-<2> If you want to use the full distribution, remove the `-slim` postfix. See xref:getting-started:editions.adoc#full-and-slim-distributions[Distributions] to learn about the differences between full and slim distributions.
+. Navigate to the folder in which you extracted the DMT package
+. Open the _migration_config/source/hazelcast.yaml_ file in your favorite editor
+. Update the `cluster-name` field to match the name of your source cluster
+. Update the `cluster-members` field to match the addresses of the cluster members
+. Save the file
+
+To update the data structure information, complete the following steps:
+
+. Navigate to the folder in which you extracted the DMT package
+. Open the _migration_config/data/imap_names.txt_ and/or the _migration_config/data/replicated_map_names.txt_ file in your favorite editor
+. Update the file content to match the names of your maps
++
+NOTE: If you have multiple data structures, use a new line for each map name.
 
-The cluster tries to start on port `5901` if it's available. It chooses another port otherwise. You can check the port in the cluster logs shown on the terminal.
+. Save the file
 
-The configuration file for the target cluster is `target.yaml`, which can be found in the folder where you extracted the DMT package.
-In case the cluster starts on a different port as you notice within the cluster logs, you must change the `address` field in `target.yaml` accordingly to match the port number.
+=== Check the Target Cluster
 
-NOTE: Instead of Docker, you can also by https://hazelcast.com/get-started/download/[download] Hazelcast 5.3.2 or newer, request a xref:getting-started:enterprise-overview.adoc#step-2-get-your-enterprise-license[trial license], and xref:getting-started:get-started-binary.adoc#step-2-start-a-local-member[start] a cluster.
+Ensure that the target cluster is running on one of the following:
 
-=== Step 4. Start the Migration Cluster
+* Enterprise Edition version 5.3.2 or later 
```

Comment:
5.3.2 or before will throw exception. Not sure if we should explicitly say that

Reply:
So, do we want to say EE version 5.3.3 or later?

If we do, that avoids having to explicitly say anything about earlier versions

Reply:
No, sorry. I mean 5.3.1 and before will throw. ok we don't have to say that then.